definitions/tools/vcf_readcount_annotator.cwl

```diff
@@ -7,7 +7,7 @@ label: "add bam_readcount info to vcf"
 baseCommand: ["vcf-readcount-annotator"]
 requirements:
     - class: DockerRequirement
-      dockerPull: "mgibio/vcf_annotation_tools-cwl:1.4.6"
+      dockerPull: "mgibio/vcf_annotation_tools-cwl:2.0.0"
```

Comment:
Should we go ahead and update to `3.0.0`?

Reply:
Done.